scripts/storybook-prep.js

```diff
@@ -85,3 +85,6 @@ fs.copyFile(HCBCssVariables, '.storybook/static/HCB_css_variables.css', (err) =>
 fs.copyFile(HCWCssVariables, '.storybook/static/HCW_css_variables.css', (err) => {
     if (err) throw err;
 });
+fs.copyFile('./dist/semantic-elements.css', '.storybook/static/semantic-elements.css', (err) => {
```

Comment:
If you update with master, we got rid of this step and set it up to just be referenced like `./semantic-elements.css'` https://github.com/SAP/fundamental-styles/blob/master/.storybook/preview.js#L12